app/views/partials/theme/simple/layout/header.liquid

```diff
@@ -1,57 +1,56 @@
 <header class="py-4">
-  <div class="container flex flex-col flex-wrap items-center mx-auto md:flex-row">
-    <a class="flex items-center font-medium text-gray-900" href="/">
-      <svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" stroke-linecap="round"
-        stroke-linejoin="round" stroke-width="2" class="w-10 h-10 p-2 text-white bg-blue-500 rounded-full"
-        viewBox="0 0 24 24">
-        <path d="M12 2L2 7l10 5 10-5-10-5zM2 17l10 5 10-5M2 12l10 5 10-5"></path>
-      </svg>
-      <span class="ml-3 text-xl">{{ 'app.title' | t }}</span>
-    </a>
+    <div class="container flex flex-col flex-wrap items-center mx-auto md:flex-row">
 
-    {% include 'theme/simple/home/quick_search_form' %}
+        <a class="flex items-center font-medium text-gray-900" href="/">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="w-10 h-10 p-2 text-white bg-blue-500 rounded-full" viewBox="0 0 24 24">
+                <path d="M12 2L2 7l10 5 10-5-10-5zM2 17l10 5 10-5M2 12l10 5 10-5"></path>
+            </svg>
+            <span class="ml-3 text-xl">{{ 'app.title' | t }}</span>
+        </a>
 
-    <nav class="flex flex-wrap items-center justify-center text-base">
-      <a href="/dashboard/items/new" class="p-3 hover:text-gray-900">{{ 'app.header.add' | t }}</a>
-      <a href="/search" class="p-3 hover:text-gray-900">{{ 'app.items.list' | t }}</a>
-      <a href="/groups" class="p-3 hover:text-gray-900">{{ 'app.groups.list' | t }}</a>
+        {% include 'theme/simple/home/quick_search_form' %}
 
-      {% if context.current_user.id %}
-        {% assign superadmins = context.constants.superadmins | split: ',' %}
-        {% if superadmins contains context.current_user.email %}
-          <a href="/admin" class="p-3 hover:text-gray-900">{{ 'app.admin' | t }}</a>
-        {% endif %}
-        <a href="/dashboard" class="p-3 hover:text-gray-900">{{ 'app.header.feed' | t }}</a>
-        <a href="/inbox" class="p-3 hover:text-gray-900" id='inbox-notifications' data-current-user-id="{{ profile.id }}" data-current-user-slug="{{ profile.slug }}">
-          <p>{{ 'app.header.chat' | t }}</p>
-          <span id="notificationsBell" class="relative inline-flex rounded-full h-3 w-3 bg-pink-500 hidden"></span>
-        </a>
+        <nav class="flex flex-wrap items-center text-base mx-auto mt-4 lg:mx-0 lg:mt-0">
+            <a href="/dashboard/items/new" class="p-3 hover:text-gray-900">{{ 'app.header.add' | t }}</a>
+            <a href="/search" class="p-3 hover:text-gray-900">{{ 'app.items.list' | t }}</a>
+            <a href="/groups" class="p-3 hover:text-gray-900">{{ 'app.groups.list' | t }}</a>
+
+            {% if context.current_user.id %}
+                {% assign superadmins = context.constants.superadmins | split: ',' %}
```

Comment:
@godot @piotrze we might want to think how to extract this somehow to be already available in the theme ; the less logic in the theme the better, thoughts?

Reply:
OK
https://trello.com/c/wF5UGIYr